### DISCUSSION TITLE
how to vertical align fields (Checkbox in my case) when the form has multiple fields on row ?

### DISCUSSION THREAD

**Discussion by quasiperfect** [filamentphp/filament#10748] [Help] [answered]
hi
i have to fields on the same row and one is a checkbox
![image](https://github.com/filamentphp/filament/assets/287513/d0ad2b98-e32d-45a3-864e-012b05c7a274)
how can i vertical align the checkbox field so the layout will look better ?

**Comment by MNhat0308** [comment]
same here! i want this too any solution

**Comment by MNhat0308** [comment, marked as answer]
i found this . maybe it will help you
![Screenshot 2024-01-11 at 14 32 15](https://github.com/filamentphp/filament/assets/57012342/89d44ece-0735-4a25-bfa3-8ac02d02bab2)
create a class in app.css
and in your php file add like this
![Screenshot 2024-01-11 at 14 33 10](https://github.com/filamentphp/filament/assets/57012342/c19e727d-a64e-4311-a3fe-9e5816d41448)
and finally your result is
![image](https://github.com/filamentphp/filament/assets/57012342/97a983fb-8266-43b6-9b80-a3b01b0ff31b)

**Reply by quasiperfect** [reply]
thanks @MNhat0308 i will try tonight and come back with feedback

**Reply by quasiperfect** [reply]
not exactly what i wanted but the function `extraAttributes` will allow me to find something that works

**Reply by jazzy1986** [reply]
i think extraAttributes not able to move the label down
